### PR TITLE
Exposes IsReverseDims in BlockInfo

### PR DESCRIPTION
**Pull request by williamfgc**
related to request in #1553 from @berkgeveci

### CODE REVIEW

File: bindings/CXX11/adios2/cxx11/Variable.h [hunk 1]
```diff
@@ -348,6 +348,8 @@ class Variable
         size_t Step = 0;
         /** reference to internal block data (used by inline Engine) */
         const T *Data() const;
+        /** true: Dims were swapped from column-major, false: not swapped */
```
Comment (germasch) [inline bindings/CXX11/adios2/cxx11/Variable.h]: minor comment: dims may also have been swapped from row-major to col-major, when reading from Fortran.

Comment (williamfgc) [reply bindings/CXX11/adios2/cxx11/Variable.h]: @germasch this particular struct is only exposed in the C++11 bindings

**Comment by williamfgc** [comment]
@germasch see my comment in #1553, it's not unreasonable to think we might need this info on a per block basis. The block inspection functions in C++ were designed with in-situ viz as one of the use-cases. It's very much needed.

**Comment by chuckatkins** [comment]
@williamfgc please edit the PR to target `master` instead.  If we want to put out a patch release soon after with this then we can.  For now though, given the short time frame, let's keep it out of the release.

**Comment by williamfgc** [comment]
@chuckatkins fine with me

**Comment by williamfgc** [comment]
request in #1553

**Pull request merged**